_posts/en/newsletters/2021-06-02-newsletter.md

```diff
@@ -0,0 +1,124 @@
+
+- FIXME:harding hwi
+
+- [LND 0.13.0-beta.rc3][LND 0.13.0-beta] is a release candidate that
+  adds support for using a pruned Bitcoin full node, allows receiving
+  and sending payments using Atomic MultiPath ([AMP][topic multipath payments]),
+  and increases its [PSBT][topic psbt] capabilities, among other improvements
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20833][] is the first PR in an effort to bring [package
+  relay][topic package relay] to Bitcoin Core. This change allows the
+  `testmempoolaccept` RPC to accept multiple transactions. Future PRs will
+  enable submitting transaction packages to the mempool through the RPC as well as
+  specifications for communicating packages over the P2P network.
+
+- [Bitcoin Core #22017][] Update Windows code signing certificate FIXME:jonatack
+
+- [Bitcoin Core #18418][] wallet: Increase OUTPUT_GROUP_MAX_ENTRIES to 100 FIXME:Xekyo
+
+- [C-Lightning #4501][] adds [JSON schemas][] for the output of the
+  roughly half of C-Lightning's current commands (with schemas for the
+  other half planned to be added in the future).  Output produced during
+  a run of C-Lightning's test suite is validated against the schemas to
+  ensure consistency.  The schemas are also used to automatically
+  generate C-Lightning's documentation about what output each command
+  produces.
+
+- [LND #5025][] adds basic support for using [signet][topic signet].  Of
+  the other LN implementations tracked by Optech, C-Lightning also has
+  support for signet (see [Newsletter #117][news117 cl4068]).
+
+- [LND #5155][] adds a configuration option to randomly select which
+  wallet UTXOs to spend in a transaction; this reduces UTXO
+  fragmentation in the wallet over time.  By contrast, the default coin
+  selection algorithm in LND spends higher value UTXOs before lower
+  value UTXOs; this minimizes fees in the short term but may result in
+  needing to pay higher fees in the future when all inputs near the size
+  of a transaction, or larger, have already been spent.
+
+- [BOLTs #672][] BOLT 2: `option_shutdown_anysegwit` FIXME:dongcarl
+
+- [BOLTs #872][] updates [BOLT3][] to specify in more detail the sort order
+  to use for commitment transaction inputs and outputs.  This was
+  originally specified by reference to [BIP69][], but one commentator
+  points out that has caused three separate problems that lead to
+  accidental channel closures and small amounts of funds lost to
+  unnecessary onchain fees.  The commetator suggests that this is
```

Comment:
The "originally specified" (past tense) -> "points out"/"suggests" (present tense) feels a little awkward to me.